src/Infrastructure/Search/SearchableContentUpdater.php

```diff
@@ -83,10 +89,24 @@ private function buildSearchableContent(
         $parts = [
             $rawView->title()->asString($languageCode),
             $rawView->subtitle()->asString($languageCode),
-            $rawView->content()->about()->description()->asString($languageCode),
             ...array_values($rawView->keywords()->asArrayOfStrings($languageCode)->getArrayCopy()),
         ];
 
         return implode(' ', $parts);
     }
+
+    /**
+     * @psalm-param LanguageCodes $languageCode
+     */
+    private function buildSearchableContentExtended(
+        DegreeProgramViewRaw $rawView,
+        string $languageCode
+    ): string {
+
+        $parts = [
+            $rawView->content()->about()->description()->asString($languageCode),
+        ];
```

Comment:
I was weird from the first look to creating an additional meta field, which is just a copy of the content field. But after checking, I agree it decreases coupling, and "content" fields can be easily extended in the future.